### PR TITLE
BBD: Assist cards

### PR DESCRIPTION
**Pull request by paulsnoops**
Scripts for the remaining Assist cards. Needs #4986. Charging Binox already in that PR.

- [x] Bloodborn Scoundrels
- [x] Bring Down
- [x] Dwarven Lightsmith
- [x] Fan Favorite
- [x] Game Plan
- [x] Gang Up
- [x] Huddle Up
- [x] Lava-Field Overlord
- [x] Magma Hellion
- [x] Out of Bounds
- [x] Play of the Game
- [x] Skystreamer
- [x] Spellweaver Duo
- [x] The Crowd Goes Wild
- [x] Vampire Charmseeker

### CODE REVIEW

File: forge-gui/res/cardsfolder/g/golden_argosy.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Types:Legendary Artifact Vehicle
 PT:3/6
 T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigFlicker | TriggerDescription$ Whenever CARDNAME attacks, exile each creature that crewed it this turn. Return them to the battlefield tapped under their owner's control at the beginning of the next end step.
 SVar:TrigFlicker:DB$ ChangeZoneAll | ChangeType$ Creature.CrewedThisTurn | Origin$ Battlefield | Destination$ Exile | RememberLKI$ True | SubAbility$ DelTrig
-SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | RememberObjects$ RememberedLKI | SubAbility$ DBCleanup | TriggerDescription$ Return them to the battlefield tapped under their owner's control at the beginning of the next end step.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/g/golden_argosy.txt:7]: The cleanup should be added

File: forge-gui/res/cardsfolder/p/play_of_the_game.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Name:Play of the Game
+ManaCost:6 W W
+Types:Sorcery
+K:Assist
+A:SP$ ChangeZoneAll | Cost$ 6 W W | UseAllOriginZones$ True | Origin$ Battlefield | Destination$ Exile | ChangeType$ Permanent.nonLand | IsCurse$ True | SpellDescription$ Exile all nonland permanents.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/p/play_of_the_game.txt:5]: Useall wrong

File: forge-gui/res/cardsfolder/g/golden_argosy.txt [hunk 3]
```diff
@@ -4,7 +4,7 @@ Types:Legendary Artifact Vehicle
...
+SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | RememberObjects$ RememberedLKI | TriggerDescription$ Return them to the battlefield tapped under their owner's control at the beginning of the next end step.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/g/golden_argosy.txt:7]: I could be wrong, but I think you want to add a DBCleanup line with ClearRemembered

File: forge-gui/res/cardsfolder/g/gang_up.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Name:Gang Up
+ManaCost:X B
+Types:Instant
+K:Assist
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/g/gang_up.txt:4]: @tehdiplomat will your current Assist wiring properly handle this {X} in the reminder text?

Comment (tehdiplomat) [reply forge-gui/res/cardsfolder/g/gang_up.txt:4]: We can test it. But i'm pretty sure Announce happens first so it should be fine.

File: forge-gui/res/cardsfolder/t/the_crowd_goes_wild.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:The Crowd Goes Wild
+ManaCost:X G
+Types:Sorcery
+K:Assist
+A:SP$ PutCounter | CounterType$ P1P1 | TargetMin$ 0 | TargetMax$ X | ValidTgts$ Creature | TgtPrompt$ Select up to X target creatures | SubAbility$ DBTrample | SpellDescription$ Support X. (Put a +1/+1 counter on each of up to X target creatures.)
+SVar:DBTrample:DB$ PumpAll | ValidCards$ Creature.counters_GE1_P1P1 | KW$ Trample | SpellDescription$ Each creature with a +1/+1 counter on it gains trample until end of turn.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/t/the_crowd_goes_wild.txt:6]: @Northmoc how should I correctly display the Oracle on these lines? It currently shows like this but the card images has them on new lines:

image

File: forge-gui/res/cardsfolder/g/gang_up.txt (hunk 4 above)
Comment (paulsnoops) [reply forge-gui/res/cardsfolder/g/gang_up.txt:4]: X works fine on my testing

File: forge-gui/res/cardsfolder/t/the_crowd_goes_wild.txt (hunk 5 above)
Comment (tehdiplomat) [reply forge-gui/res/cardsfolder/t/the_crowd_goes_wild.txt:6]: You could try putting a \n before the word Each, but also its not something we really worry about consistently.

Comment (paulsnoops) [reply forge-gui/res/cardsfolder/t/the_crowd_goes_wild.txt:6]: \n didn't work for that line but if it's not a big deal we can merge it like this and sort it another time

**Review by tool4ever — APPROVED** [review]
cheers

**Pull request merged**